src/main/java/luffy/LuffyUI.java

```diff
@@ -0,0 +1,128 @@
+package luffy;
+import java.util.Scanner;
+
+import static java.lang.String.format;
+
+
+/**
+ * Represents a UI for the chatbot that deals with
+ * user interaction
+ */
+public class LuffyUI {
+
+    private final Scanner commandScanner;
+
+    public LuffyUI() {
+        this.commandScanner = new Scanner(System.in);
+    }
+
+    public void showLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * This method prints out a welcome message when the chatbot is started
+     */
+    public void showWelcomeMessage() {
+        showLine();
+        System.out.println(" Hello! I'm luffy.Luffy");
```

Comment:
Perhaps a possible typo?

```suggestion
        System.out.println(" Hello! I'm Luffy");
```

Reply:
^ I had the same problem when refactoring it into packages and classes as IDEs might automatically search for name occurence and modify then during the process.